test/integration/Highlight.e2e.test.js

```diff
@@ -52,4 +52,42 @@ describe('Highlights', () => {
         // Assert highlight creation mode is not active
         cy.getByTestId('bp-AnnotationsControls-highlightBtn').should('not.have.class', 'is-active');
     });
+
+    it('should restrict highlight annotation to single page', () => {
+        // Show the preview
+        cy.showPreview(Cypress.env('FILE_ID_DOC'));
+
+        // Wait for the empty highlight layer to be present
+        cy.getByTestId('ba-Layer--highlight');
+
+        // Select texts across pages
+        cy.get('.page')
+            .first()
+            .find('.textLayer')
+            .children()
+            .last()
+            .as('firstPageEl')
```

Comment:
Is this naming accurate? It looks like this is aliasing the first child of the textLayer?